app/assets/javascripts/components/reviews/review_form.js.jsx

```diff
@@ -0,0 +1,27 @@
+var ReviewForm = React.createClass({
+
+  getInitialState: function(){
+    return { film: this.props.film, content: ''}
+  },
+  handleContentChange: function(e){
+    this.setState({ content: e.target.value })
+  },
+  handleSubmit: function(e){
+    e.preventDefault();
+    var data = {content: this.state.content.trim(), filmId: this.state.film.id }
+    this.props.onReviewSubmit(data)
+    this.setState({ content:'' })
```

Comment:
I would not place `content` in `state`. This is data that does not need be tracked.

The content could be cleared by setting the DOM element's value to empty string, so all this is unnecessary.